docs/operations/faq/overview.rst

```diff
@@ -0,0 +1,12 @@
+.. _faq_overview:
+
+Envoy FAQ
+==========
+
+The purpose of this document is to cover commonly used Envoy configuration and scenarios in typical
+Envoy deployment, e.g., how to setup zone aware routing. 
+
+.. toctree::
```

Comment:
This page is going to get too long. Move each FAQ answer to its own RST file. Also, figure out a way where the TOC on this page is actually visible inline.